concordium-consensus/src/Concordium/Queries.hs

```diff
@@ -1090,8 +1091,14 @@ getAccountInfoHelper getASI getCooldowns acct bs = do
         aiAccountAddress <- BS.getAccountCanonicalAddress acc
         aiAccountCooldowns <- getCooldowns acc
         aiAccountAvailableAmount <- BS.getAccountAvailableAmount acc
-        -- TODO(drsk). github #1225. Implement getAccountIsSuspended
-        let aiAccountIsSuspended = False
+        aiAccountIsSuspended <- do
+            mAccBaker <- BS.getAccountBaker acc
+            case mAccBaker of
+                -- If the account doesn't have an associated validator, then it can't be suspended.
+                Nothing -> return False
+                Just accBaker ->
+                    return $
+                        fromCondDef (_bieAccountIsSuspended $ _accountBakerInfo accBaker) False
```

Comment:
@td202 This will need a change to `concordium-base` and as well to `concordium-grpc-api` to remove the flag there from `AccountInfo`. Should we do this first? Or as a follow up?

Reply:
Personally I would love if we could do this as a follow up task.

Reply:
Either way is fine.

Reply:
Ok, then let me merge this and change this as a follow up. The PR's for the change in base is already there: https://github.com/Concordium/concordium-base/pull/564.